src/pages/home/report/ReportActionItem.js

```diff
@@ -164,7 +164,7 @@ class ReportActionItem extends Component {
             >
                 <Hoverable resetsOnClickOutside>
                     {hovered => (
-                        <View>
+                        <>
```

Comment:
**question:** Why are we removing this `View`?

Reply:
Seems unnecessary.

Reply:
Please revert.